Formula/l/lsr.rb

```diff
@@ -0,0 +1,33 @@
+class Lsr < Formula
+  desc "Ls but with io_uring"
+  homepage "https://tangled.sh/@rockorager.dev/lsr"
+  url "https://tangled.sh/@rockorager.dev/lsr",
+      using:    :git,
+      tag:      "v1.0.0",
+      revision: "9bfcae0be1d3ee2db176bb8001c0f46650484249"
+  license "MIT"
+
+  depends_on "zig" => :build
+  depends_on :macos
```

Comment:
Why does this require macos only?

Reply:
During build or testing? I think access to `io_uring` requires sudo on Linux.

Reply:
During testing. That might be a problem, do you think should I skip test for linux?

Reply:
On linux, could test for the permission error.